package.json

```diff
@@ -27,7 +27,8 @@
   "dependencies": {
     "@actions/core": "^1.10.1",
     "@aws-sdk/client-codedeploy": "^3.598.0",
-    "@aws-sdk/client-ecs": "^3.594.0",
+    "@aws-sdk/client-ecs": "^3.620.0",
+    "aws-sdk": "^2.1664.0",
```

Comment:
Curious why we need to import the `v3` and `v2` versions of the SDK?

Reply:
Ideally, the unit tests should also be running off of `v3`, does anything fail if you remove the dependency?